testing/stubs.go

```diff
@@ -109,6 +110,10 @@ func TestDecisionInterceptor(testID string, stubbedWorkflows, stubbedShortWorkfl
 					d.ScheduleActivityTaskDecisionAttributes.TaskList = &swf.TaskList{Name: S(*d.ScheduleActivityTaskDecisionAttributes.TaskList.Name + testID)}
 				}
 			}
+
+			if interceptor != nil {
+				interceptor.AfterDecision(decision, ctx, outcome)
```

Comment:
After considering the functions (and the activity interceptors and more in the future),we probably should go with a composed approach. I put something together that should do the trick:
https://github.com/sclasen/swfsm/pull/64